server/models/item.ts

```diff
@@ -0,0 +1,86 @@
+import { Model, Transaction } from 'objection';
+
+export interface IItemCreateInput {
+  id: string;
+  name: string;
+  pokemonId: string;
+  price: number;
+  happiness: number;
+  imageUrl: string;
+}
+
+export interface IItemEditInput {
+  name?: string;
+  pokemonId?: string;
+  price?: number;
+  happiness?: number;
+  imageUrl?: string;
+}
+
+export default class Item extends Model {
+  static pickJsonSchemaProperties = true;
+  static tableName = 'item';
+
+  static jsonSchema = {
+    type: 'object',
+    properties: {
+      id: { type: 'string', format: 'uuid' },
+      name: { type: 'string' },
+      pokemonId: { type: 'string', format: 'uuid' },
+      price: { type: 'number' },
+      happiness: { type: 'number' },
+      imageUrl: { type: 'imageUrl' },
+      createdAt: { type: 'string' },
+      updatedAt: { type: 'string' },
+      deletedAt: { type: ['string', 'null'] },
+    },
+    required: ['name', 'pokemonId', 'price', 'happiness', 'imageUrl'],
+  };
+
+  static async create(input: IItemCreateInput, txn: Transaction) {
```

Comment:
```suggestion
  static async create(input: IItemCreateInput, txn: Transaction): Promise<Item> {
```
we do this with all model methods (and most functions in general) as a helpful annotation for anyone who wants to use the function and to ensure that if you make changes to the function you're not accidentally changing the return type